portal-web/docroot/html/taglib/ui/form_navigator/lexicon/sections.jspf

```diff
@@ -113,27 +113,26 @@
 				</c:otherwise>
 			</c:choose>
 
-			Liferay.once(
-				'<portlet:namespace />formReady',
-				function(event) {
-					if (!sectionContent.hasClass('in')) {
-						if (focusField.length) {
+			if (focusField.length) {
```

Comment:
The check has to happen because in one case we fire it after the ```show.bs.collapse``` event, and in another case, we fire it right away.